CONTRIBUTING.md

```diff
@@ -1,27 +1,40 @@
 # Contributing Guidelines
 
-Thank you for your interest in contributing to the OpenSource documentation! Whether it's a bug report, new feature, correction, or additional
-documentation, we greatly value feedback and contributions from our community.
+Thank you for your interest in improving the OpenSource documentation! We value and appreciate all feedback and contributions from our community, whether it’s a documentation request, bug report, or a correction to existing content. 
 
-Please read through this document before submitting any issues or pull requests to ensure we have all the necessary
-information to effectively respond to your bug report or contribution.
+It’s important for us to tell you that the OpenSearch team is dedicated to providing complete and best-in-class documentation. We realize that we are a bit behind! Please bear with us as we are building our documentation team and content library.  We appreciate your support and patience! 
 
 
-## Reporting Bugs/Feature Requests
+## How to request changes
 
-Use the GitHub issue tracker to report bugs or suggest features.
+Before entering a change request, please read this document carefully. We want to make sure we have all the information necessary to effectively respond to your contribution. 
 
-When filing an issue, please check existing open, or recently closed, issues to make sure somebody else hasn't already
-reported the issue. Please try to include as much information as you can. Details like these are incredibly useful:
+There are two ways to contribute: create an issue describing your request or create a pull request that can be directly injected into the content.
 
-* A reproducible test case or series of steps
-* The version of our code being used
-* Any modifications you've made relevant to the bug
-* Anything unusual about your environment or deployment
+*Create an Issue*
 
+Use the Github issue tracker to request additional documentation, suggest content corrections, and report technical issues with the documentation site. 
 
-## Contributing via Pull Requests
-Contributions via pull requests are much appreciated. Before sending us a pull request, please ensure that:
+1. Go to https://github.com/opensearch-project/documentation-website/issues and select *New issue*.
+2. Enter the requested information and include as much detail as possible, especially which version the request affects.
+3. Select *Submit new issue*. 
+
+
+The ‘untriaged’ label will be assigned automatically. During the triage process, we will add the appropriate labels, assign the issue to a technical writer, and prioritize the request. We may follow up with you for additional information. 
+
+
+*Create a pull request*
```

Comment:
```suggestion
*Create a pull request*

```

Reply:
Not done yet.... 
Please wait to review. Thanks!

Reply:
oh sorry. it went thru email as a request for review. maybe mark as draft.